### PR TITLE
httping: add page

### PR DESCRIPTION
**Pull request by jyrodgers**
cross-platform software to measure the latency and throughput of a webserver

homepage: https://www.vanheusden.com/httping/
man page: https://docs.oracle.com/cd/E86824_01/html/E54763/httping-1.html

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/httping.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# httping
+
+> Measures the latency of a webserver.
+
+- Measure latency of webserver using a URL:
+
+`httping -g {{URL}}`
```
Comment (sbrl) [inline pages/common/httping.md]: Perhaps a concrete example ehre would work better?

File: pages/common/httping.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Measure latency of webserver using a hostname:
```
Comment (sbrl) [inline pages/common/httping.md]: Perhaps ....`latency of a webserver`.... would read better?

File: pages/common/httping.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`httping -h {{hostname}}`
+
+- Measure latency using a specific port:
+
+`httping {{webserver}} -p {{port}}`
```
Comment (sbrl) [inline pages/common/httping.md]: Perhaps a concrete example of `{{webserver}}` would work better?

File: pages/common/httping.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+- Measure latency only a specific number of times:
```
Comment (agnivade) [inline pages/common/httping.md]: Does this repeat the test multiple times (the default being 1) ? If so, I would suggest - "Repeat the test a given number of times"

File: pages/common/httping.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+`httping -c {{count}} {{webserver}}`
+
+- Measure latency using SSL (for this to work you need to give a 'https'-url or a 443 portnumber):
+
+`httping -l {{webserver}}`
+
+- Measure latency using authentication:
```
Comment (agnivade) [inline pages/common/httping.md]: "Use basic auth mechanism with a given username and password"

**Comment by jyrodgers** [comment]
Thanks for the suggestions. I've added these changes.

**Comment by jyrodgers** [comment]
My example failed the CI test. What is the best preferred way to include examples?

**Comment by agnivade** [comment]
Please see - https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md. In short, we use concrete values in tokens rather than generic values.

**Pull request closed without merging**